src/control/controlindicator.h

```diff
@@ -17,24 +22,24 @@ class ControlIndicator : public ControlObject {
     ControlIndicator(const ConfigKey& key);
     virtual ~ControlIndicator();
 
-    void setBlinkValue(enum BlinkValue bv);
+    void setBlinkValue(enum BlinkValue newBlinkValue);
 
   signals:
     void blinkValueChanged();
 
   private slots:
-    void slotGuiTick50ms(double cpuTime);
     void slotBlinkValueChanged();
+    void set(double value) {
```

Comment:
Overwriting a non-virtual function from the parent is confusing. Please rename as `slotValueChanged` and then simply invoke `set()`.

Reply:
Done,